src/main/java/com/example/bookstoreapp/servive/impl/BookServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.example.bookstoreapp.servive.impl;
+
+import com.example.bookstoreapp.model.Book;
+import com.example.bookstoreapp.repository.BookRepository;
+import com.example.bookstoreapp.servive.BookService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookServiceImpl implements BookService {
+    @Autowired
+    private BookRepository bookRepository;
```

Comment:
let's use constructor injection instead so that it is guaranteed that our service will be fully initialized